drake/Makefile

```diff
@@ -33,13 +33,9 @@ endif
 
 .PHONY: clean
 clean:
-	cmake --build pod-build --target clean
-# for windows cmd shell
-	@-rd /s pod-build        
-	@-rd /s build
-# for everyone else
-	@-rm -rf pod-build
-	@-rm -rf build
+	cmake --build pod-build --target clean-all
```

Comment:
absolutely right.  drake-distro/Makefile needs clean-all, but the rest just clean (a copy and paste bug).  sigh.  i'll fix it.  good catch!